src/test/java/uk/gov/digital/ho/hocs/workflow/processes/MPAM.java

```diff
@@ -102,7 +105,7 @@ public void happyPath() {
                 .execute();
 
         verify(mpamProcess)
-                .hasCompleted("CallActivity_1c4zy60");
+                .hasCompleted("CallActivity_0g8kpaf");
```

Comment:
Worth associating this in a variable at the top, would make it clearer what this CallActivity does?